lib/api/src/grpc/proto/points.proto

```diff
@@ -118,6 +118,7 @@ message SetPayloadPoints {
   optional PointsSelector points_selector = 5; // Affected points
   optional WriteOrdering ordering = 6; // Write ordering guarantees
   optional ShardKeySelector shard_key_selector = 7; // Option for custom sharding to specify used shard keys
+  optional string key = 8; // Option for indicate property of payload
```

Comment:
Would need to come up with better description for this field. (note mostly to myself)